pod-network-chaos/prow_run.sh

```diff
@@ -31,7 +31,7 @@ envsubst < pod_network_scenario.yaml.template > pod_network_scenario.yaml
 export SCENARIO_FILE=pod-network-chaos/pod_network_scenario.yaml
 envsubst < config.yaml.template > pod_network_scenario_config.yaml
```

Comment:
you need to create the file at the tmp location and export the scenarios file with the proper location like: 

https://github.com/redhat-chaos/prow-scripts/blob/dc69d428f2ed142c9ecf6a74e1900652fbe70d80/pvc-scenario/prow_run.sh#L22C51-L22C76